draft-ietf-quic-version-negotiation.md

```diff
@@ -387,6 +387,41 @@ lifetime of the connection; endpoints MUST NOT change the version that they
 consider to be in use based on the Version field of long headers as that field
 could be forged by attackers.
 
+# Minimum support
+
+Hosts can support this standard incrementally to simplify implementation, if
+not all features of the design are desired.
+
+{{INV}} and {{QUIC}} already describe how servers send Version Negotiation
+packets in response to long header packets with unsupported version fields, even
+if they support only one version. Doing so prevents long timeouts at the client,
+but servers might not send the packet multiple times for the same connection, or
+when under stress.
```

Comment:
It's a summary of Sec 6.1 of RFC9000. The usual case would be an INITIAL with 0RTT packets -- if the server can recognize the packet type, it can limit the number of packets to 1 statelessly.  But one can certainly imagine some sort of global state saying "I am under DoS attack, stop sending VN". 

The important thing this paragraph is saying is reminding the reader that other RFCs say that you might get a VN packet, but you might not. I'm happy to get a suggested rewording.

Reply:
Well, the server is free to not respond to *any* message it wants. What's unique about VN.